test/object_store.jl

```diff
@@ -55,6 +55,15 @@
         yield()
         @test local_count(oid) == 0
     end
+
+    @testset "Object owner" begin
+        obj = Ray.put(1)
+        # ownership only embedded in ObjectRef on serialization
+        obj_rt = Ray.deserialize_from_ray_object(Ray.serialize_to_ray_object(obj))
+        addr_json = ray_jll.MessageToJsonString(Ray.get_owner_address(obj))
+        addr_rt_json = ray_jll.MessageToJsonString(obj_rt.owner_address)
+        @test addr_json == addr_rt_json == obj_rt.owner_address_json
+    end
```

Comment:
Something is weird here. I think it's the comment about "ownership only embedded in ObjectRef on serialization" and the immediately using the non-serialized `obj` and fetching it's address

Reply:
Maybe show that `obj.owner_address` is `nothing`?

Reply:
main goal here was to get coverage for some code that was not covered before (the `getproperty` overload), so we need to get an object that has embedded `owner_address_json` that's _not_ nothing.

Reply:
however, taht code is now being exercised in the actual `deserialize` path so it might not be a big deal to just 🔪 this test suite...